packages/demo/usecase-demo/src/components/Alert.tsx

```diff
@@ -16,18 +16,24 @@ export function Alert({
   return (
     <div
       className={cn(
-        'flex items-center gap-x-5 rounded-md border p-5 font-medium',
+        'flex items-center gap-x-5 rounded-lg border p-5 font-medium',
         variant === 'success'
           ? 'border-green-300 bg-success text-success-foreground'
           : variant === 'info'
             ? 'border-blue-300 bg-blue-50 text-blue-400'
-            : 'border-orange-300 bg-danger text-danger-foreground',
+            : 'border-[#9A1D1D] bg-[#FBCFCF] text-[#9A1D1D]',
```

Comment:
Quite hard to read: hard to distinguish all three possibilities.
I'll try to make something more readable with 'class-variance-authority', like in `toast.tsx`.

Reply:
Hard-coded HEX colors: Did you find these values in our Figma?
Can we define them in `index.css`, instead of those two values that seem to be unused:
```
    --danger: var(--orange-50);
    --danger-foreground: var(--orange-500);
```